cc7-reana/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM centos:centos7
+
+# Install all required packages
+RUN yum install -y http://linuxsoft.cern.ch/wlcg/centos7/x86_64/wlcg-repo-1.0.0-1.el7.noarch.rpm && \
+    yum install -y HEP_OSlibs git environment-modules which                                      && \
+    yum clean all                                                                                && \
+    curl -o /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py                                 && \
+    chmod +x /tmp/get-pip.py                                                                     && \
+    /tmp/get-pip.py                                                                              && \
+    rm -f /tmp/get-pip.py                                                                        && \
+    pip install PyYAML requests                                                                  && \
+    mkdir -p /opt/ali-bot                                                                        && \
+    cd /opt/ali-bot                                                                              && \
+    git clone https://github.com/alisw/ali-bot /opt/ali-bot
```

Comment:
Speaking with the preservation hat on, is there some known `ali-bot` version released that could be cloned here, so that the creation of Docker images could be reproducible even several months later into the future?